decidim-verifications/app/views/decidim/verifications/authorizations/first_login.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="text-center py-10">
-  <h1 class="h3 md:h3 md:h1 decorator inline-block text-left mb-12">
+  <h1 class="h3 md:title-decorator inline-block text-left mb-12">
```

Comment:
By reading it, I am not very sure about this change.

Reply:
This is showing that we don't have a linter for Tailwind in the pipeline. I'll check out some alternatives so we can detect these inconsistencies automatically. 

As far as I see, I couldn't find this particular problem (two md:h* classes in the same element), only on this one: 

```shell
$ grep -ri "md:h.*md:h" decidim*/app/views/
decidim-verifications/app/views/decidim/verifications/authorizations/first_login.html.erb:  <h1 class="h3 md:h3 md:h1 decorator inline-block text-left mb-12">
``` 

But that doesn't mean that we don't have others elements with same utilities classes with this same or similar problem.